src/ristretto/bulletproofs_plus.rs

```diff
@@ -157,7 +156,7 @@ impl BulletproofsPlusService {
     /// Helper function to prepare a batch of private range statements
     pub fn prepare_private_range_statements(
         &self,
-        statements: Vec<&RistrettoAggregatedPrivateStatement>,
+        statements: &Vec<&RistrettoAggregatedPrivateStatement>,
```

Comment:
nit: References to Vec force callers to use a Vec instead of anything that dereferences to a slice, with no benefit. https://rust-lang.github.io/rust-clippy/v0.0.212/index.html#ptr_arg

```suggestion
        statements: &[&RistrettoAggregatedPrivateStatement],
```

Reply:
Yeah, good point. I'd made that fix elsewhere at the linter's suggestion, but must have missed it here. I'll take another look at the other associated PRs as well.